### PR TITLE
new deploy to azure china graphics

### PR DESCRIPTION
**Pull request by dicolanl**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* added new png and svg files for deploy to china

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@dicolanl - what's the use case for this?  since none of the samples link to the China cloud?

**Comment by dicolanl** [comment]
@bmoore-msft Microsoft Sentinel GitHub (https://github.com/Azure/Azure-Sentinel) uses the deploy to azure buttons.  There is a need for deploy to Azure China.  I figured maybe someone else would need to use it too, so i put in the central place where the other buttons are.

**Comment by bmoore-msft** [comment]
Where is it consumed?

File: 1-CONTRIBUTION-GUIDE/images/deploytoazurechina.svg [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+<svg
+   width="210"
+   height="34"
+   viewBox="0 0 210 34"
+   fill="none"
+   version="1.1"
+   id="svg27"
+   sodipodi:docname="deploytoazurechina.svg"
+   inkscape:version="1.1.2 (b8e25be833, 2022-02-05)"
+   inkscape:export-filename="C:\Users\ndicola\OneDrive - Microsoft\Downloads\newlogo\deploytoazuregov.png"
```
Comment (bmoore-msft) [inline 1-CONTRIBUTION-GUIDE/images/deploytoazurechina.svg:11]: remove the export properties

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Pull request merged**